lldb/test/Shell/Stepper/Inputs/stepper.py

```diff
@@ -0,0 +1,214 @@
+from __future__ import print_function
```

Comment:
why is this in Inputs? Should it be in helper/?

Reply:
Fixed.